packages/hoprd/hopr-admin/connection.js

```diff
@@ -81,10 +91,10 @@ export class Connection {
         ? await fetch(`https://${window.location.host}/api/ssl`).then(
             (_) => new WebSocket('wss://' + window.location.host)
           )
-        : new WebSocket('ws://' + window.location.host)
+        : new WebSocket('ws://' + 'localhost:19501/')
```

Comment:
we should listen for messages and logs through:

- `/api/v2/messages/websocket`
- `/api/v2/node/stream/websocket`